src/sas_robot_driver_client.cpp

```diff
@@ -121,35 +128,40 @@ void RobotDriverClient::send_target_joint_positions(const VectorXd &target_joint
 {
     std_msgs::msg::Float64MultiArray ros_msg;
```

Comment:
@juanjqo: Maybe better to throw an exception if a blacklisted method is called. If this doesn't do anything it will be hard to debug.